gitlab.go

```diff
@@ -52,11 +53,22 @@ func (client *gitLabClient) listProjects(page int) ([]*gitlab.Project, gitLabPag
 func (client *gitLabClient) getProject(groupName string, projectName string) (*gitlab.Project, error) {
 	project, _, err := client.Projects.GetProject(fmt.Sprintf("%v/%v", groupName, projectName), nil, nil)
 	if err != nil {
-		log.Error().
+		ev := log.Error().
 			WithError(err).
 			WithString("group", groupName).
-			WithString("project", projectName).
-			Message("Failed to get project.")
+			WithString("project", projectName)
+		ev.Message("Failed to get project.")
```

Comment:
Is it possible to check if it's a 404 error in particular?

Reply:
Will look at this tomorrow.

Reply:
Oops, forgot about this, check new changes please.